### ISSUE TITLE
Add support for graphql@15.0.0

### ISSUE THREAD

**Issue by adamovittorio** [cjoudrey/graphql-schema-linter#228]
Updating peerDependency at graphql@15.0.0, cause this error:

```
✖ graphql-schema-linter:
It looks like you may have hit a bug in graphql-schema-linter.
It would be super helpful if you could report this here: https://github.com/cjoudrey/graphql-schema-linter/issues/new
TypeError: (0 , _buildASTSchema.getDescription) is not a function
    at Object.FieldDefinition (/Users/vittorio.adamo/Workspace/live-platform/node_modules/graphql-schema-linter/lib/rules/fields_have_descriptions.js:15:46)
    at Object.enter (/Users/vittorio.adamo/Workspace/live-platform/node_modules/graphql/language/visitor.js:323:29)
    at Object.enter (/Users/vittorio.adamo/Workspace/live-platform/node_modules/graphql/utilities/TypeInfo.js:370:25)
    at visit (/Users/vittorio.adamo/Workspace/live-platform/node_modules/graphql/language/visitor.js:243:26)
    at validate (/Users/vittorio.adamo/Workspace/live-platform/node_modules/graphql/validation/validate.js:69:24)
    at validateSchemaDefinition (/Users/vittorio.adamo/Workspace/live-platform/node_modules/graphql-schema-linter/lib/validator.js:66:43)
    at run (/Users/vittorio.adamo/Workspace/live-platform/node_modules/graphql-schema-linter/lib/runner.js:68:58)
```

Link to renovate pr: #222

**Comment by cjoudrey** [comment]
Hey @adamovittorio, thanks for opening the issue.

I actually started looking into bumping `graphql` to `15.0.0` in this branch: https://github.com/cjoudrey/graphql-schema-linter/compare/bump-graphqljs-15

I think I was mostly done, but never got around merging it. I'll take a look later this week.

**Comment by adamovittorio** [comment]
Hi @cjoudrey, Thank you for your work! 🙏🏻
Let me know if I can help.

**Comment by cjoudrey** [comment]
I just published `v0.3.0` with GraphQL v15 support. 😄 Cheers!
